Added documentation string to package to deprecate global nickname

NicoSimoski opened the pull request:
Hello again! :-)

Is this sufficient to close [issue #24](https://github.com/vindarel/cl-readline/issues/24)?
What could be improved?

NicoSimoski committed:
Added documentation string to package to deprecate global nickname

vindarel commented:
We need to add a note in the `docs/cl-readline.info`.

(and while we're at it, to add a real documentation sentence in the package)

BTW we can also use 

    (uiop:add-package-local-nickname :rl :cl-readline)

with the right ASDF version.

NicoSimoski commented:
I will do that, but I think I won't have time next week.

Isn't 
`(uiop:add-package-local-nickname :rl :cl-readline)`
supposed to be used at runtime when you don't want do update your `defpackage` form?

vindarel commented:
mmh while only runtime? I agree updating the defpackage form is the clean way, this uiop function is best when testing a package while in the REPL, but it could be added on top of a file too. It's worth documenting the two anyways.

NicoSimoski committed:
Updated package documentation string

- added description of package
- added option UIOP:ADD-PACKAGE-LOCAL-NICKNAME for local nicknames

NicoSimoski commented:
Hello! :-)

Sorry for the delay: I extended my easter vacation.

I updated the documentation string of the package.
Which kind of note would you like to have in doc/cl-readline.texi?
Other than the use of :local-nicknames in the example(s), I mean.

NicoSimoski committed:
Added local-nickname to examples in cl-readline.texi

vindarel commented:
Thank you again! Wish you had good vacations.

NicoSimoski commented:
My pleasure and thanks for your time and advice! :-)

I was helping out family, so they were pleased but I'm exhausted now. :-D
